CMakeLists.txt

```diff
@@ -439,6 +439,7 @@ add_executable(SystemTestsRunner
     "source/tests/system/nirfsa_driver_api_tests.cpp"
     "source/tests/system/nirfsg_driver_api_tests.cpp"
     "source/tests/system/nirfsg_session_tests.cpp"
+    "source/tests/system/nirfmxinstr_driver_api_tests.cpp"
     "source/tests/system/nirfmxspecan_session_tests.cpp"
```

Comment:
Nitpick: I must have missed this previously, but these aren't in alphabetical order anymore

Reply:
Same with the other places this driver was added.